examples/solvers/bregman_tv_tomography.py

```diff
@@ -0,0 +1,91 @@
+"""Example of Bregman-TV in tomography.
+
+The example does five iterations in a Bregman-TV scheme, applied to a
+tomography problem.
+"""
+
+import odl
+import numpy as np
+
+# Discrete reconstruction space
+reco_space = odl.uniform_discr(min_pt=[-20, -20], max_pt=[20, 20],
+                               shape=[300, 300], dtype='float32')
+
+# Make a parallel beam geometry with flat detector
+angle_partition = odl.uniform_partition(0, np.pi, 100)
+detector_partition = odl.uniform_partition(-30, 30, 558)
+geometry = odl.tomo.Parallel2dGeometry(angle_partition, detector_partition)
+
+# Create the ray transform
+ray_trafo = odl.tomo.RayTransform(reco_space, geometry, impl='astra_cuda')
+
+
+# Create phantom
+discr_phantom = odl.phantom.shepp_logan(reco_space, modified=True)
+
+# Create sinogram of forward projected phantom with noise
+noise_free_data = ray_trafo(discr_phantom)
+noise = odl.phantom.white_noise(ray_trafo.range)
+noise = noise * 1/noise.norm() * noise_free_data.norm() * 0.10
```

Comment:
small thing, but why not `noise *= (1/noise.norm() * noise_free_data.norm() * 0.10)`?